yesod-test/Yesod/Test.hs

```diff
@@ -693,6 +694,25 @@ get url = request $ do
     setMethod "GET"
     setUrl url
 
+-- | Follow a redirect, if the last response was a redirect.
+-- ==== __Examples__
```

Comment:
There needs to be an empty haddock line before `-- ==== __Examples__`; it currently looks like this:
![image](https://cloud.githubusercontent.com/assets/1274145/13934298/b76f25fa-ef6e-11e5-9aa3-6a03bb183239.png)